src/genie/libs/parser/viptela/show_cloudexpress_applications.py

```diff
@@ -1,13 +1,11 @@
 from genie.metaparser import MetaParser
```

Comment:
what happened here? There shouldnt be any modifications to this file

Reply:
User changed file for some reason. I have copied the most up to date version of this file from https://github.com/CiscoTestAutomation/genieparser/blob/master/src/genie/libs/parser/viptela/show_cloudexpress_applications.py

There should be no differences between the show_cloudexpress_applications.py file in this PR and the file in the external master branch